include/zephyr/drivers/espi/espi.h

```diff
@@ -134,29 +134,29 @@ enum espi_pc_event {
  * @cond INTERNAL_HIDDEN
  *
  */
-#define ESPI_PERIPHERAL_INDEX_0  0ul
-#define ESPI_PERIPHERAL_INDEX_1  1ul
-#define ESPI_PERIPHERAL_INDEX_2  2ul
+#define ESPI_PERIPHERAL_INDEX_0 0ul
+#define ESPI_PERIPHERAL_INDEX_1 1ul
+#define ESPI_PERIPHERAL_INDEX_2 2ul
```

Comment:
nit: for these and below they should really use fixed width constants `ul` can mean different things, prefer `UINT64_C(0)`, `UINT64_C(1)`, `UINT64_C(2)`, etc over `ul`

Reply:
I've only reformatted this with clang-format, but I will change this. Good to know about these macros, didn't know them.